package.json

```diff
@@ -40,6 +41,7 @@
     "@uiw/react-md-editor": "^3.20.1",
     "bcryptjs": "^2.4.3",
     "date-fns": "^2.29.2",
+    "dotenv-flow": "^3.2.0",
```

Comment:
Do we need `dotenv-flow`. Couldn't this be done with just `dotenv`? I'm asking because this introduces an entirely new dependency, while `dotenv` is already required by both `remix` and `prisma`. We would still need to add it as a dependency here, but it would only mark it as a direct dependency on the package-lock.json file, since it is already there.

Oooooh, I also just remembered, that blitz did use `dotenv-flow`, for the

> ability to have multiple .env* files like .env.development, .env.test and .env.production

But on this repo we are not using those files anymore, and we only have a single `.env` file right now.

Reply:
Yes that is what this dependency is for, if we are fully controlling the environment (dev, prod, etc) via GitHub actions i think we could get rid off this dependency. What do you think?